src/lcov.jl

```diff
@@ -73,4 +76,68 @@ function writeline(io::IO, line::Int, count::Nothing)
     (0, 0)
 end
 
+
+"""
+    readfile(infofile::AbstractString) -> Vector{FileCoverage}
+
+Read coverage data from a file in LCOV info format.
+"""
+function readfile(infofile::AbstractString)
+    source_files = FileCoverage[]
+    coverage = nothing
+    for line in eachline(infofile)
+        if startswith(line, "end_of_record")
+            coverage = nothing
+        elseif (m = match(r"^SF:(.+)", line)) !== nothing
+            sf = String(m[1])
+            coverage = FileCoverage(sf, "", CovCount[])
+            push!(source_files, coverage)
+        elseif (m = match(r"^DA:(\d+),(-?\d+)(,[^,\s]+)?", line)) !== nothing
+            if coverage !== nothing
+                ln = parse(Int, m[1])
+                da = parse(Int, m[2])
+                cv = coverage.coverage
+                lc = length(cv)
+                if ln > 0
+                    if lc < ln
+                        resize!(cv, ln)
+                        cv[(lc + 1):ln] .= nothing
+                    end
+                    cv[ln] = something(cv[ln], 0) + da
+                end
+            end
+        end
+        nothing
+    end
+    return source_files
+end
+
+"""
+    readfolder(folder) -> Vector{FileCoverage}
```

Comment:
Nitpick, but: should it be `read_file` and `read_folder` to match `process_file` and `process_folder`?

Reply:
I was matching the existing `LCOV.writefile` with the name `LCOV.readfile`, and the general base guidelines to avoid underscores. I agree it would be nice to be more consistent, though I'm not sure how to accomplish that.

Reply:
OK, I see -- yeah, when two differing "conventions" clash, this is always a bit nasty. In any case, it doesn't matter much, I was just wondering.